File: 1769-minimum-number-of-operations-to-move-all-balls-to-each-box/1769-minimum-number-of-operations-to-move-all-balls-to-each-box.cpp
class Solution {
public:
    vector<int> minOperations(string boxes) {
        int n = boxes.size();
        vector<int>ans(n,0);
        for(int i=0; i<n; i++){
            if(boxes[i]=='1'){
                for(int j=0; j<n; j++){
                    ans[j]+=abs(j-i);
                }
            }
        }
        return ans;
    }
};

File: 1769-minimum-number-of-operations-to-move-all-balls-to-each-box/1769-minimum-number-of-operations-to-move-all-balls-to-each-box.py
class Solution:
    def minOperations(self, boxes: str) -> list[int]:
        n = len(boxes)
        ans = [0] * n
        for i in range(n):
            if boxes[i] == '1':
                for j in range(n):
                    ans[j] += abs(j - i)
        return ans
